core/http-auth-aws/src/main/java/software/amazon/awssdk/http/auth/aws/crt/internal/signer/DefaultAwsCrtV4aHttpSigner.java

```diff
@@ -64,9 +68,10 @@ private static V4aProperties v4aProperties(BaseSignRequest<?, ? extends AwsCrede
         Clock signingClock = request.requireProperty(SIGNING_CLOCK, Clock.systemUTC());
         Instant signingInstant = signingClock.instant();
         AwsCredentialsIdentity credentials = sanitizeCredentials(request.identity());
-        String regionName = request.requireProperty(REGION_NAME);
+        Set<RegionScope> regionSet = request.requireProperty(REGION_SET, Collections.singleton(RegionScope.GLOBAL));
```

Comment:
I set the default here of global for example, but if we decide no default, then this should be removed.